### DISCUSSION TITLE
[REQUEST] groq whisper API for more accurate real time transcription

### DISCUSSION THREAD

**Discussion by magiasdu** [Marksdo/Whisper#41] [Ideas]
Awesome app! I've bought it last week, but something missing that would make it even more useful is adding support for using groq whisper API for real time transcription, deepgram is not good for accurate french, is this possible for you?

**Comment by Marksdo** [comment]
Thank you very much for your suggestion and for supporting our development work through your purchase. 

The integration of the Whisper API had been conducted before June 2024
Plans:
- real-time voice transcription 
- regular project transcription
- Regenerate voice from transcripts. 

However, during debugging and development, OpenAI banned my API account after June 2024 (as I live in a region that falls under their newly enforced restriction policies - 2024.6).

For now, I am unable to obtain a new account. 

Please wait for 1-2 more versions, and I will prioritize this integration.
